packages/mobile/src/verify/VerificationFailedModal.tsx

```diff
@@ -1,24 +1,30 @@
-import TextButton from '@celo/react-components/components/TextButton'
-import colors from '@celo/react-components/styles/colors'
-import fontStyles from '@celo/react-components/styles/fonts'
-import { componentStyles } from '@celo/react-components/styles/styles'
 import * as React from 'react'
 import { useTranslation } from 'react-i18next'
-import { StyleSheet, Text, View } from 'react-native'
-import Modal from 'react-native-modal'
+import { setRetryVerificationWithForno } from 'src/account/actions'
+import { WarningModal } from 'src/components/WarningModal'
 import { Namespaces } from 'src/i18n'
 import { cancelVerification } from 'src/identity/actions'
 import { VerificationStatus } from 'src/identity/verification'
-import { navigateHome } from 'src/navigator/NavigationService'
+import { navigate, navigateHome } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { toggleFornoMode } from 'src/web3/actions'
 
 interface Props {
   verificationStatus: VerificationStatus
+  retryWithForno: boolean
+  fornoMode: boolean
   cancelVerification: typeof cancelVerification
+  setRetryVerificationWithForno: typeof setRetryVerificationWithForno
+  toggleFornoMode: typeof toggleFornoMode
 }
 
 export function VerificationFailedModal(props: Props) {
   const { t } = useTranslation(Namespaces.nuxVerification2)
-  const [isDismissed, setIsDismissed] = React.useState(false)
+  const [isDismissed, setIsDismissed] = React.useState(true)
+
+  React.useEffect(() => {
+    setIsDismissed(false)
+  }, [setIsDismissed])
```

Comment:
This change was necessary to prevent a ghost modal from showing up briefly when a user reopens the Verification loading page and the modal was dismissed. This use of useEffect is like `componentDidMount()`

Reply:
Might be nice to have this in a comment ion code :)